docs/posts/macos-ventura-privacy-security-updates.md

```diff
@@ -0,0 +1,88 @@
+
+- **Messages**: Most message attachments are blocked, besides certain image, video, and audio attachments. This includes most iMessage "apps" such as in-conversation games. Link previews are also disabled.
+- **FaceTime**: Incoming calls are blocked, unless you have previously called that person/contact. This is likely in response to past FaceTime bugs, such as the [exploit in 2019](https://9to5mac.com/2019/01/28/facetime-bug-hear-audio/) which allowed an attacker to listen in on your microphone before you picked up the call.
+- **Photos**: The new Shared Albums functionality in iCloud Photos is blocked, and invitations are automatically rejected. Shared Albums do still work on any devices without Lockdown Mode enabled.
+
+In addition to these, other Apple services like **Home** will reject incoming invitations unless you have previously interacted with the sender.
+
+### Device Changes
+
+Some other device functionality is limited with Lockdown Mode enabled as well. If you have a Mac with Apple Silicon, connected devices/accessories are not allowed to connect unless your Mac is unlocked and explicit approval of the device is given. Configuration profiles can no longer be installed either, and the ability to enroll in a Mobile Device Management (MDM) system is disabled. These are enterprise management features, which are occasionally abused to control or monitor devices.
+
+## Passkeys
+
+**Passkeys** are likely to be the most impactful new feature for the everyday person's security practices. Passkeys are a cross-platform standard supported by Google, Apple, and Microsoft. Based on the FIDO2 standard, passkeys are the first real effort to replace passwords as your primary mode of authentication.
+
+Using a passkey stored on your phone is supported by most browsers in macOS, but only Safari currently allows you to use a passkey stored on your Mac at this time. Passkeys generated on your iPhone or Mac are stored in iCloud Keychain, which is end-to-end encrypted with your phone and/or computer's lock screen password. In Safari, passkeys are replacing the single-device WebAuthn registration, which allowed you to use your computer as a hardware key with Touch ID.
+
+The benefits of passkeys are fairly limited at the moment, until more services support the standard for primary or multi-factor authentication.
+
+## Rapid Security Responses
+
+Alongside iOS 16, Apple is introducing **Rapid Security Responses** (RSR) to macOS Ventura. RSR allows Apple to ship small security fixes on the fly much faster than before.
+
+Previously, releasing even a small security patch meant Apple has to release a huge multi-gigabyte update package. This was required to preserve the system's cryptographic integrity following changes to how the system volume is handled in macOS Big Sur. With RSR, updates are much smaller, meaning patches can be downloaded much quicker and applied more reliably.
```

Comment:
```suggestion
Previously, releasing even a small security patch meant Apple had to release a huge multi-gigabyte update package. This was required to preserve the system's cryptographic integrity, following changes to how the system volume is handled in macOS Big Sur. With RSR, updates are much smaller, so patches can be downloaded much faster and applied more reliably.
```

Corrects tense disagreement.
Adds stylistic comma for subordinate clause.
Amends word choice for readability.